package.json

```diff
@@ -8,11 +8,11 @@
     "version": "1.1.0"
   },
   "scripts": {
-    "start": "node build/main.js --run",
-    "dev": "node build/main.js --run-dev",
-    "package": "node build/main.js --package",
-    "test": "nyc -s node build/main.js --test",
-    "lint": "nyc node build/main.js --lint",
+    "start": "NODE_ENV=\"production\" node build/main.js --run",
+    "dev": "NODE_ENV=\"development\" node build/main.js --run-dev",
+    "package": "NODE_ENV=\"production\" node build/main.js --package",
+    "test": "NODE_ENV=\"production\" nyc -s node build/main.js --test",
+    "lint": "NODE_ENV=\"production\" nyc -s node build/main.js --lint",
```

Comment:
Sadly for all involved this sort of environment variable setting doesn't work on Windows.